### PR TITLE
Convert `web/api/document` to markdown for es

### LINKED ISSUE #8664: chore: convert web/api/document for Spanish (es) to Markdown
As part of the larger 100% Markdown [project](https://github.com/mdn/translated-content/issues/7486), I am converting the set of documents under web/api/document to Markdown.

[Tracking spreadsheet](https://docs.google.com/spreadsheets/d/1Mk3bQoHXozN2rUWn9rxVHA5MiO0whDLuj71KKC0iMSg/edit#gid=617080625)

### PR DESCRIPTION
**Pull request by jakul**
I applied some content fixes to the HTML files pre-conversion and some more to the markdown files post-conversion. 

Fixes https://github.com/mdn/translated-content/issues/8664.

### CODE REVIEW

File: files/es/web/api/document/cookie/index.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+  - `;path=path` (p. ej.: '/'. '/midir'). Si no se especifica, por defecto corresponde a la ruta del documento actual.
+    La ruta debe ser **absoluta** (ver [RFC 6265](https://tools.ietf.org/html/rfc6265)). Para más información sobre cómo utilizar rutas relativas, ir a [este párrafo](/es/docs/DOM/document.cookie$edit#Using_relative_URLs_in_the_path_parameter).
+  - `;domain=domain` (p. ej. 'example.com', 'subdomain.example.com'). Si no se especifica, su valor por defecto es la porción de la dirección web de la ubicación actual del archivo. A diferencia de lo que ocurría en las primeras especificaciones, los puntos iniciales de los nombre de dominio se ignoran, pero los navegadores pueden impedir crear cookies que contengan dichos puntos. Si se especifica un dominio, los subdominios siempre son incluidos.
+  - `;max-age=`_`duración-máxima-en-segundos`_ Por ejemplo: 60\*60\*24\*365 para un año.
+  - `;expires=`\_`fecha-en-formato-GMTString` \_Si no se especifica `max-age` ni `expires`, la cookie expirará al terminar la sesión actual.
```
Comment (Graywolf9) [inline files/es/web/api/document/cookie/index.md:39]:
```suggestion
  - `;max-age=duración-máxima-en-segundos` Por ejemplo: 60\*60\*24\*365 para un año.
  - `;expires=fecha-en-formato-GMTString` Si no se especifica `max-age` ni `expires`, la cookie expirará al terminar la sesión actual.
```

File: files/es/web/api/document/cookie/index.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+document.cookie = "comida_preferida=tripa";
+function alertCookie() {
+  alert(document.cookie); // visualizar: nombre=oeschger;comida favorita=tripa
+
+}
+```
+
+```js
+<button onclick="alertCookie()">Mostrar cookies</button>
+```
```
Comment (Graywolf9) [inline files/es/web/api/document/cookie/index.md:69]:
````suggestion
```html
<button onclick="alertCookie()">Mostrar cookies</button>
````
```

File: files/es/web/api/document/cookie/index.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+var cookieValor = document.cookie.replace(/(?:(?:^|.*;\s*)test2\s*\=\s*([^;]*).*$)|^.*$/, "$1");
+
+function alertCookieValue() {
+  alert(cookieValor);
+}
+```
+
+```js
+<button onclick="alertCookieValue()">Mostrar valor de cookie</button>
+```
```
Comment (Graywolf9) [inline files/es/web/api/document/cookie/index.md:88]:
````suggestion
```html
<button onclick="alertCookieValue()">Mostrar valor de cookie</button>
```
````

File: files/es/web/api/document/cookie/index.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  if (document.cookie.replace(/(?:(?:^|.*;\s*)hacerAlgoUnaSolaVez\s*\=\s*([^;]*).*$)|^.*$/, "$1") !== "true") {
+    alert("Hacer algo aquí!");
+    document.cookie = "hacerAlgoUnaSolaVez=true; expires=Fri, 31 Dec 9999 23:59:59 GMT";
+  }
+}
+```
+
+```js
+<button onclick="dhacerUnaVez()">Solo hacer algo una vez</button>
+```
```
Comment (Graywolf9) [inline files/es/web/api/document/cookie/index.md:107]:
````suggestion
```html
<button onclick="dhacerUnaVez()">Solo hacer algo una vez</button>
```
````

File: files/es/web/api/document/createdocumentfragment/index.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+{{Specifications}}
+
+## Compatibilidad del Navegador
+
+{{Compat}}
+
+## Compatibilidad del Navegador
+
+{{Compat}}
```
Comment (Graywolf9) [inline files/es/web/api/document/createdocumentfragment/index.md:72]:
```suggestion
## Compatibilidad con navegadores

{{Compat}}
```

File: files/es/web/api/document/write/index.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  - Documentación
+  - Referencia
+  - metodo
+translation_of: Web/API/Document/write
+---
+{{ ApiRef("DOM") }}
+
+Escribe una cadena de texto dentro del hilo de un `document `abierto por [document.open()](/es/docs/Web/API/document.open).
+
+> **Nota:** Nota: dado que `document.write` escribe directo al hilo **(stream**) de un documento, la llamada a `document.write` en un documento ya cargado automáticamente ejecuta `document.open`, [lo cual limpiará todo el contenido del documento en cuestión](/es/docs/Web/API/document.open#Notes).
```
Comment (Graywolf9) [inline files/es/web/api/document/write/index.md:16]:
```suggestion
> **Nota:** dado que `document.write` escribe directo al hilo **(stream**) de un documento, la llamada a `document.write` en un documento ya cargado automáticamente ejecuta `document.open`, [lo cual limpiará todo el contenido del documento en cuestión](/es/docs/Web/API/document.open#Notes).
```

File: files/es/web/api/document/writeln/index.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
+---
+title: Document.writeln()
+slug: Web/API/Document/writeln
+translation_of: Web/API/Document/writeln
+---
+{{ ApiRef("DOM") }}
+
+Escribe una cadena de texto en el documento, seguida de una nueva línea.
+
+> **Nota:** Nota: dado que `document.writeln` (al igual que `document.write)` escribe directo al hilo (`stream`) de un documento, la llamada a `document.write` en un documento ya cargado automáticamente ejecuta `document.open`, [lo cual limpiará todo el contenido del documento en cuestión](/es/docs/Web/API/document.open#Notes).
```
Comment (Graywolf9) [inline files/es/web/api/document/writeln/index.md:10]:
```suggestion
> **Nota:** Dado que `document.writeln` (al igual que `document.write)` escribe directo al hilo (`stream`) de un documento, la llamada a `document.write` en un documento ya cargado automáticamente ejecuta `document.open`, [lo cual limpiará todo el contenido del documento en cuestión](/es/docs/Web/API/document.open#Notes).
```

**Review by Graywolf9 — APPROVED** [review]
Great job, thank you so much for your time and contribution. I have some notes:

- In the `es` localization we have some Convention on translations and one of them are "Compatibilidad con navegadores" for that section [`es` guide notes](https://github.com/mdn/translated-content/tree/main/docs/es)
- the indentation for all cases is 2 spaces as in the contribution code guidelines
- finally, I notice that some lists items have not the correct line breaks

Also, there are our chats:
- Telegram: https://t.me/+Dr6qKQCAepw4MjFj
- Matrix: https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org

**Comment by jakul** [comment]
> Great job, thank you so much for your time and contribution. I have some notes:
> 
> * In the `es` localization we have some Convention on translations and one of them are "Compatibilidad con navegadores" for that section [`es` guide notes](https://github.com/mdn/translated-content/tree/main/docs/es)
> * the indentation for all cases is 2 spaces as in the contribution code guidelines
> * finally, I notice that some lists items have not the correct line breaks
> 
> Also, there are our chats:
> 
> * Telegram: https://t.me/+Dr6qKQCAepw4MjFj
> * Matrix: https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org

Thanks for this, I was a bit unsure where to get guidance. I'll have a read through before I do the next batch of files.

**Pull request merged**